lib/firmata.js

```diff
@@ -686,11 +687,15 @@ function Board(port, options, callback) {
             this.pins.push({supportedModes: [], analogChannel: analogChannel});
           }
         }
-        ready();
+        setTimeout(function() {
+          ready();
+        }, settings.firstWriteTimeout);
```

Comment:
This should not be necessary. If there is an issue here, it is likely a larger issue and adding a timeout here is a hack around it.